targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_KL27Z/device/TOOLCHAIN_ARM_STD/MKL27Z64xxx4.sct

```diff
@@ -51,7 +51,7 @@
 #define __ram_vector_table__            1
 
 #if (defined(__ram_vector_table__))
-  #define __ram_vector_table_size__    0x00000200
```

Comment:
It has been reduced to match the number of NVIC_VECTORS, we had reserved a fixed space as this varies by platform. My request was to go back to the original value as it helps minimize changes in this file. I can check with the internal SDK team if this space can be adjusted per platform.